### DISCUSSION TITLE
Group by case sensitive

### DISCUSSION THREAD

**Discussion by fooo** [ravendb/ravendb#17808] [General]
Working with RavenDb 6 in c#.
Case sensitive queries are possible e.g using _exact: _true_ in the WHERE.
Now I also want to do a case sensitive GROUP BY. 

`GroupBy(i => i.Prop, StringComparer.Ordinal)`

doesn't seem to work. Any suggestions?

**Comment by ayende** [comment]
By default, group by _is_ case sensitive.
You don't need to do anything to enable that.
If you need it to be case insensitive, you need to `GroupBy(x=>x.Prop.ToLowerInvariant())`
